x-pack/plugins/security/public/authentication/login/login_page.tsx

```diff
@@ -235,17 +233,19 @@ export class LoginPage extends Component<Props, State> {
       );
     }
 
-    const query = parse(window.location.href, true).query;
+    const { searchParams } = new URL(window.location.href);
+
     return (
       <LoginForm
         http={this.props.http}
         notifications={this.props.notifications}
         selector={selector}
-        // @ts-expect-error Map.get is ok with getting `undefined`
-        message={messageMap.get(query[LOGOUT_REASON_QUERY_STRING_PARAMETER]?.toString())}
+        message={
+          loginFormMessages[searchParams.get(LOGOUT_REASON_QUERY_STRING_PARAMETER) as LogoutReason]
+        }
         loginAssistanceMessage={this.props.loginAssistanceMessage}
         loginHelp={loginHelp}
-        authProviderHint={query[AUTH_PROVIDER_HINT_QUERY_STRING_PARAMETER]?.toString()}
+        authProviderHint={searchParams.get(AUTH_PROVIDER_HINT_QUERY_STRING_PARAMETER) || undefined}
```

Comment:
🙌  🙌  🙌

Reply:
Oh, @thomheymann I am guessing this change is what caused the async chunks to increase:

> #### Async chunks
> > Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app
> 
> id	[before](https://github.com/elastic/kibana/commit/a33ae63025be42ad6326945ecd882721833c3c6c)	[after](https://github.com/elastic/kibana/commit/fab0034e3196a75946438d89638ab11f1bfec62a)	diff
> `security`	486.8KB	514.4KB	+27.6KB

I'm a bit on the fence, we eventually want to get rid of our usage of the `url` module completely, but maybe we should take this change out until we can change URL parsing in the rest of the plugin (see `is_internal_url.ts` and `parse_next.ts`). WDYT?

Reply:
That's strange, if anything I'd expect the bundle size to go down since we're not including url module anymore.